data/json/recipes/recipe_deconstruction.json

```diff
@@ -5021,5 +5021,14 @@
     "time": "12 s",
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "plastic_chunk", 2 ] ], [ [ "splinter", 3 ] ] ]
+  },
+{
+    "result": "icecream_scoop",
+    "type": "uncraft",
+    "activity_level": "MODERATE_EXERCISE",
+    "skill_used": "fabrication",
+    "time": "12 s",
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "components": [ [ [ "spoon", 1 ] ], [ [ "spring", 2 ] ] ]
```

Comment:
`"spring"` is supposed to be a 3 kg heavy-duty spring, so you probably won't get it from an icecream scoop.

Reply:
wow, that's an oversight on my part. That's a good point and it would make sense to remove that then. Thank you.

Reply:
Springs have been removed from the deconstruct recipe.